app/tag-management/remove-sub-tags/remove-sub-tags.component.css

```diff
@@ -0,0 +1,100 @@
+h3 {
```

Comment:
do not alter headlines. We want to be consistent on all our sub-sites, so headlines are defined in the overall css file `hip-theme`